From ff6d4326f839321c079d60ae9301c6202942f675 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 27 Aug 2016 14:31:04 +0200
Subject: [PATCH] Include <stdint.h> where needed

The {u,}int{8,16,32}_t types are defined in <stdint.h>, so it should be
included when such types are used.

Not including <stdint.h> might work by accident with some C libraries
due to it being included by other headers, but it for example causes
build failures with the musl C library.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Upstream-status: https://github.com/mdrjr/c2_aml_libs/pull/1
---
 amadec/audio-dec.h               | 1 +
 amavutils/include/Amvideoutils.h | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/amadec/audio-dec.h b/amadec/audio-dec.h
index 4325fe4..ac0f36b 100644
--- a/amadec/audio-dec.h
+++ b/amadec/audio-dec.h
@@ -12,6 +12,7 @@
 #define AUDIO_DEC_H
 
 #include<pthread.h>
+#include <stdint.h>
 
 #include <audio-out.h>
 #include <audiodsp.h>
diff --git a/amavutils/include/Amvideoutils.h b/amavutils/include/Amvideoutils.h
index 6f978e4..94f10a6 100644
--- a/amavutils/include/Amvideoutils.h
+++ b/amavutils/include/Amvideoutils.h
@@ -5,6 +5,8 @@
 extern "C" {
 #endif
 
+#include <stdint.h>
+
 #define HDMI_HDCP_PASS           (1)
 #define HDMI_HDCP_FAILED      (0)
 #define HDMI_NOCONNECT        (-1)
-- 
2.7.4

